conf/application.conf

```diff
@@ -75,45 +75,47 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.MicroserviceModule"
 play.http.filters = "uk.gov.hmrc.play.bootstrap.filters.MicroserviceFilters"
 
 # Json error handler
-play.http.errorHandler = "uk.gov.hmrc.hello.ErrorHandler"
+play.http.errorHandler = "utils.ErrorHandler"
 
 # Play Modules
 # ~~~~
 # Additional play modules can be added here
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
+play.modules.enabled += "config.DIModule"
 
 # Controller
 # ~~~~~
 # By default all controllers will have authorisation, logging and
 # auditing (transaction monitoring) enabled.
 # The below controllers are the default exceptions to this rule.
 
-controllers {
-  com.kenshoo.play.metrics.MetricsController = {
-    needsAuth = false
-    needsLogging = false
-    needsAuditing = false
-  }
-
-  uk.gov.hmrc.hello.controllers.Documentation = {
-    needsAuth = false
-    needsLogging = true
-    needsAuditing = false
-  }
-
-  uk.gov.hmrc.hello.controllers.SandboxController = {
-    needsAuth = false
-    needsLogging = true
-    needsAuditing = false
-  }
-
-  uk.gov.hmrc.hello.controllers.LiveController = {
-    needsAuth = false
-    needsLogging = true
-    needsAuditing = false
-  }
-
-}
+//controllers {
```

Comment:
Why is this all commented out? Is it not needed?

Reply:
Good question. Does not appear to be used.